src/controllers/authors.js

```diff
@@ -0,0 +1,26 @@
+const {
+    getAllItems,
+    createItem,
+    updateItem,
+    getItemById,
+    deleteItem,
+} = require('./helper-fn-CRUD-operations');
+
+const getAllAuthors = (_, res) => getAllItems(res, 'author');
+
+const createAuthor = (req, res) => createItem(res, 'author', req.body);
+
+const updateAuthor = (req, res) =>
+updateItem(res, 'author', req.body, req.params.id);
```

Comment:
If youre spreading a declaration across multiple lines, only the opening line should have the least level of indentation. As before run the linter over this an the problem is sorted for you
```
const updateAuthor = (req, res) =>
    updateItem(res, 'author', req.body, req.params.id);
```